code/src/Consumption.js

```diff
@@ -0,0 +1,30 @@
+import React from "react"
+ 
+const consumptionOptions = [
+    "read the printed book",
+    "read the e-book",
+    "listen to the audio book"
+]
+
```

Comment:
good use of semantics! 
Also great that you are using map and an array, it makes it easy to add more options!